src/nvim/misc1.c

```diff
@@ -2927,12 +2930,11 @@ char_u *vim_getenv(char_u *name, int *mustfree)
     return p;
   }
 
-  vimruntime = (STRCMP(name, "VIMRUNTIME") == 0);
```

Comment:
This is using STRCMP() and I've kept it but _setenv uses STRICMP - shouldn't this be case insensitive too?